public/styles/App.scss

```diff
@@ -3,24 +3,27 @@
 @import url('https://fonts.googleapis.com/css?family=Sunflower:300');
 
 $heading-font-family: 'Playfair Display', sans-serif;
+$body-content-font: 'Noto Serif', sans-serif;
 $header-background-image: linear-gradient(to right bottom, 
     rgba(0, 0, 0, 0.5), 
     rgba(0, 0, 0, 0.5)), 
     url('../images/banner/landing.jpeg');
+
 $orange-color:  #E47E08;
 $red-color: red;
 $icon-grey: #BEBEBE;
 $white-color: #fff;
 $white-grey: #eee;
-$facebook-blue: #4267b2;
+$facebook-blue: rgb(42, 84, 168);
 $hover-black: rgba(0,0,0,0.2);
 
 body {
     padding: 0;
     margin: 0;
-    font-family: $heading-font-family;
-    font-family: 'Noto Serif', sans-serif;
+    font-family: $body-content-font;
```

Comment:
Does this take the P/O's feedback about fonts into consideration?

Reply:
@seyi-adeleke, yes it does.